servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategies.java

```diff
@@ -131,11 +115,6 @@ public static HttpExecutionStrategy difference(final Executor fallback,
         if (noOffloads(left)) {
```

Comment:
`final Executor fallback,` parameter is no longer used

Reply:
removed